README.md

```diff
@@ -87,17 +107,99 @@ fn read_config() -> Result<(), Report> {
+format, which additionally captures a [`backtrace::Backtrace`]:
+
+![short report format](https://raw.githubusercontent.com/yaahc/color-eyre/master/pictures/short.png)
+
+<br>
 
-![short report format](./pictures/short.png)
+Finally, running `RUST_LIB_BACKTRACE=full cargo run --example usage` tells `color-eyre` to use
+the full format, which in addition to the above will attempt to include source lines where the
+error originated from, assuming it can find them on the disk.
 
-## Full Report Format (with `RUST_LIB_BACKTRACE=full`)
+![full report format](https://raw.githubusercontent.com/yaahc/color-eyre/master/pictures/full.png)
 
-![full report format](./pictures/full.png)
+### Custom `Section`s for error reports via [`Help`] trait
+
+The `section` module provides helpers for adding extra sections to error reports. Sections are
+disinct from error messages and are displayed independently from the chain of errors. Take this
+example of adding sections to contain `stderr` and `stdout` from a failed command, taken from
+[`examples/custom_section.rs`]:
+
+```rust
+use color_eyre::{SectionExt, Help, Report};
+use eyre::eyre;
+use std::process::Command;
+use tracing::instrument;
+
+trait Output {
+    fn output2(&mut self) -> Result<String, Report>;
+}
+
+impl Output for Command {
+    #[instrument]
+    fn output2(&mut self) -> Result<String, Report> {
+        let output = self.output()?;
+
+        let stdout = String::from_utf8_lossy(&output.stdout);
+
+        if !output.status.success() {
+            let stderr = String::from_utf8_lossy(&output.stderr);
+            Err(eyre!("cmd exited with non-zero status code"))
+                .with_section(move || {
+                    "Stdout:"
+                        .skip_if(|| stdout.is_empty())
+                        .body(stdout.trim().to_string())
+                })
+                .with_section(move || {
+                    "Stderr:"
+                        .skip_if(|| stderr.is_empty())
+                        .body(stderr.trim().to_string())
+                })
+        } else {
+            Ok(stdout.into())
+        }
+    }
+}
+```
+
+---
+
+Here we have an function that, if the command exits unsuccessfully, creates a report indicating
+the failure and attaches two sections, one for `stdout` and one for `stderr`. Each section
+includes a short header and a body that contains the actual output. Additionally these sections
+use `skip_if` to tell the report not to include them if there was no output, preventing empty
+sections from polluting the end report.
+
+Running `cargo run --example custom_section` shows us how these sections are included in the
+output:
+
+![custom section example](https://raw.githubusercontent.com/yaahc/color-eyre/master/pictures/custom_section.png)
+
+As you can see, only the `Stderr:` section actually gets included, because the `cat` command
+fails stdout ends up being empty and is skipped in the final report. Giving us a short and
```

Comment:
nit: I would drop "As you can see". Also I can't, the link is 404'ing for me (but I think that's because this branch isn't merged?

Do you need a comma between "because the `cat` command fails" and "stdout ends up being empty"?

The last sentence feels incomplete; "giving us" suggests a continuation of something existing, not a standalone sentence. How about "This gives us"?